test/unit/common/build-region.spec.js

```diff
@@ -1,294 +1,355 @@
 
           it('uses the el', function() {
-            expect(this.region.el).to.equal(this.el);
+            expect(region.el).to.equal(el);
           });
 
           describe('with `parentEl` also defined', function() {
+            let parentEl;
+
             beforeEach(function() {
-              this.parentEl = $('<div id="not-actual-parent"></div>');
-              this.definition = _.defaults({parentEl: this.parentEl}, this.definition);
-              this.region = this.view.addRegion(_.uniqueId('region_'),this.definition);
+              parentEl = $('<div id="not-actual-parent"></div>');
+              definition = _.defaults({parentEl: parentEl}, definition);
+              region = view.addRegion(_.uniqueId('region_'), definition);
             });
 
             it('returns the jQuery(el)', function() {
-              expect(this.region.getEl(this.el)).to.deep.equal($(this.el));
+              expect(region.getEl(el)).to.deep.equal($(el));
             });
 
           });
         });
 
         describe('when el is a jQuery object', function() {
+          let el;
+          let definition;
+          let region;
+
           beforeEach(function() {
-            this.el = $('<div id="baz-region">');
-            this.bazRegion = new this.DefaultRegionClass({el: this.el});
-            this.definition = {el: this.el};
-            this.region = this.view.addRegion(_.uniqueId('region_'),this.definition);
+            el = $('<div id="baz-region">');
+            new DefaultRegionClass({el: el});
+            definition = {el: el};
+            region = view.addRegion(_.uniqueId('region_'),definition);
           });
 
           it('uses the default region class', function() {
-            expect(this.region).to.be.an.instanceof(this.DefaultRegionClass);
+            expect(region).to.be.an.instanceof(DefaultRegionClass);
           });
 
           it('uses the el', function() {
-            expect(this.region.el).to.equal(this.el[0]);
+            expect(region.el).to.equal(el[0]);
           });
         });
       });
 
       describe('when el is an empty jQuery object', function() {
+        let el;
+        let definition;
+        let buildRegion;
+
         beforeEach(function() {
-          this.el = $('i-am-not-real');
-          this.definition = {el: this.el};
+          el = $('i-am-not-real');
+          definition = {el: el};
 
-          this.buildRegion = function() {
-            this.view.addRegion(_.uniqueId('region_'),this.definition);
-          }.bind(this);
+          buildRegion = function() {
+            view.addRegion(_.uniqueId('region_'),definition);
+          };
         });
 
         it('throws a `NoElError`', function() {
-          expect(this.buildRegion).to.throw('An "el" must be specified for a region.');
+          expect(buildRegion).to.throw('An "el" must be specified for a region.');
         });
       });
 
       describe('with `regionClass` defined', function() {
+        let $el;
```

Comment:
these `el` and `$el` are only used inside the next `describe` block.. let's move them there